pkg/install/go.mod

```diff
@@ -3,9 +3,16 @@ module github.com/fluxcd/flux/pkg/install
 go 1.13
 
 require (
+	github.com/Masterminds/semver v1.5.0 // indirect
+	github.com/cyphar/filepath-securejoin v0.2.2 // indirect
+	github.com/ghodss/yaml v1.0.0 // indirect
+	github.com/gobwas/glob v0.2.3 // indirect
 	github.com/instrumenta/kubeval v0.0.0-20190918223246-8d013ec9fc56
+	github.com/pkg/errors v0.9.1 // indirect
 	github.com/shurcooL/httpfs v0.0.0-20190707220628-8d4bc4ba7749
 	github.com/shurcooL/vfsgen v0.0.0-20181202132449-6a9ea43bcacd
 	github.com/stretchr/testify v1.4.0
 	golang.org/x/tools v0.0.0-20200121210457-b3205ff6fffe // indirect
+	k8s.io/apimachinery v0.17.2 // indirect
+	k8s.io/helm v2.16.1+incompatible
```

Comment:
These are the sort of dependencies we were trying to avoid with a separate install module.

Reply:
Oh no, please do not add Helm and Kubernetes back to the install package.

Reply:
To be fair, this is something that changed out from under me, and not something I *re*introduced.